husky_gazebo/launch/controller_utils.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+
+  <!-- Robot state publisher -->
+  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
+    <param name="publish_frequency" type="double" value="50.0" />
+    <param name="tf_prefix" type="string" value="" />
+  </node>
+
+ <!-- Joint state publisher -->
+  <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher"/>
```

Comment:
Without the JSP, the visualization of the Husky fails (E.g if you don't have Gazebo  launched and only have the robot_description set). I noticed this on the huskys that we're building that this is the case. Using the husky_viz spawns a husky with its wheels (& other nonfixed) being spawned without a TF and looks like a mess.

I guess the question is: how can you visualize the husky w/o a JSP?